pkg/secrethub/credentials/encoding.go

```diff
@@ -226,7 +227,7 @@ type rsaPrivateKeyDecoder struct{}
 func (d rsaPrivateKeyDecoder) Decode(payload []byte) (*RSACredential, error) {
 	key, err := x509.ParsePKCS1PrivateKey(payload)
 	if err != nil {
-		return nil, err
+		return nil, ErrMalformedCredential
```

Comment:
Consider making this an `ErrorPref` and wrapping the original error. This will help us quite a lot when debugging a support ticket.

Reply:
Good point.